client/components/App.js

```diff
@@ -27,15 +29,14 @@ class App extends React.Component {
     this.getData();
   }
 
-  //  18.191.88.165
   getData() {
-    axios.get('http://localhost:3000/reviews')
+    axios.get(`/reviews/${Math.floor(Math.random() * 10000000)}`)
```

Comment:
this will technically never get the product id 10,000,000 but rather 9,999,999 at max

Reply:
apologize i'll add a +1